### DISCUSSION TITLE
Globals Variables

### DISCUSSION THREAD

**Discussion by eternityCoder1** [webdriverio/webdriverio#6421] [Q&A] [answered]
Hi, i am currently using Nightwatchjs. I want to move my automation to WebdriverIO. I am looking for a functionality like "globals" available in NightwatchJS.

below is the settings file option:
========================
// Location of an external globals module which will be loaded and made available to the test as a
//  property globals on the main client instance.
  globals_path: <path of the file>

in test I can read/update these variables like below:
browser.globals.password

**Comment by christian-bromann** [comment, marked as answer]
Hey @eternityCoder1 as mentioned in https://webdriver.io/docs/browserobject#custom-configurations you can set your customer options in the config file and then always access it through `browser.config.someCustomOption`. Let me know if this works for you!

**Reply by eternityCoder1** [reply]
Thank you @christian-bromann for your help. Greatly appreciated. It worked.
